recipes/croaring/meta.yaml

```
@@ -0,0 +1,63 @@
+{% set name = "CRoaring" %}
+{% set version = "0.2.59" %}
+{% set sha256 = "02f80850e856001dbcd2571d140bab3d2bfb1e5bcc1e3b271d4c4e964dd65518" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/RoaringBitmap/CRoaring/archive/v0.2.59.tar.gz
```

This will make it slightly easier to keep the recipe updated.:
```suggestion
  url: https://github.com/RoaringBitmap/CRoaring/archive/v{{ version }}.tar.gz
```

---

done
thanks @chrisburr